### ISSUE TITLE
[BUG] Plugin wont load on 1.20.6

### ISSUE THREAD

**Issue by gerardvanschip** [snowgears/shopbugs#425]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Error in console

### Environment

```markdown
- Server Version: Paper 1.20.6 (#79)
- Shop Version: 1.8.4.7
```

### Server startup log

https://paste.gg/p/anonymous/f98f85d0cb304374953f353a1af8fbea

### Shop config files

https://paste.gg/p/anonymous/1677f1a394104a1dba153d9defcb8307

### Bug description

The error shows in the latest.log file

The plugin wont load. I have tried running only the shop plugin with the server jar but same result. Also deleted the config files so fresh ones would be created, same result.

### Steps to reproduce

Start server, see error in the startup log showing the plugin is being disabled.

### Expected behaviour

Plugin should load.

### Actual behaviour

Plugin gets disabled.

### Other information

None

**Comment by gerardvanschip** [comment]
Is there anything else I should add?

**Comment by matthew02** [comment]
Same versions of Paper and Shop and same error.

**Comment by gerardvanschip** [comment]
> Same versions of Paper and Shop and same error.

Is it maybe a paper thing?

**Comment by matthew02** [comment]
> > Same versions of Paper and Shop and same error.
> 
> Is it maybe a paper thing?

Maybe? I also tried 1.20.4 and Purpur (Paper fork). Same results.

**Comment by OstlerDev** [comment]
Same issue with Paper 1.21 Experimental

**Comment by ScuroK** [comment]
same here on paper. can you please fix it?

**Comment by Wullverin2** [comment]
We have the same error. Is there already a workaround or fix for this.
https://pastebin.com/URCpxc0b

**Comment by OstlerDev** [comment]
An update is being developed and tested currently. SnowGears expects to release this around the end of July or start of August. The update will support both Paper 1.20.6 and Paper 1.21.

**Comment by OstlerDev** [comment]
Fixed in 1.9.0.0 & 1.9.0.1
